e2e/fixtures/orders.js

```diff
@@ -66,6 +86,11 @@ module.exports = [
     subtype: 'Final',
     fullType: 'Final supervision order',
     document: 'final_supervision_order.pdf',
+    dateOfIssue: {
```

Comment:
Thoughts on doing one of the tests without a date of issue? Just to ensure that the autopop works as expected?

Reply:
Will do blank order without date of issue